rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/run/configurations/unityExe/UnityExeConfigurationType.kt

```diff
@@ -1,11 +1,10 @@
 package com.jetbrains.rider.plugins.unity.run.configurations.unityExe
 
 import com.intellij.execution.configurations.ConfigurationTypeBase
-import com.intellij.util.IconUtil
 import icons.UnityIcons
 
-class UnityExeConfigurationType : ConfigurationTypeBase("RunUnityExe", "Standalone Player", // "RunUnityExe" preserved for compatibility
-    "Unity Standalone Player configuration", UnityIcons.RunConfigurations.UnityExe) {
+class UnityExeConfigurationType : ConfigurationTypeBase("RunUnityExe", "Unity Executable", // "RunUnityExe" preserved for compatibility
+    "Either UnityEditor or Unity Standalone Player", UnityIcons.RunConfigurations.UnityExe) {
```

Comment:
I would be tempted to leave this as it was. We do use it internally for launching the editor if it's not already running, but we don't intend the user to use it for that purpose. As far as the user is concerned, they use the "Attach to Editor" config and that's all - they should never need to manually add a run config that launches the editor. But it is intended for use with standalone players.